### DISCUSSION TITLE
Last update custom FBW - crash sim

### DISCUSSION THREAD

**Discussion by kolyadkl** [flybywiresim/aircraft#4294] [General]
I see conflicts with last update custom FBW and scenes in my collection. When I try to load a flight, sim crashed. I tried to remove all scenes from community folder and now all ok. But I didn't add any new scenes long time and custom FBW worked fine. With all this scenes work fine default A320 and other planes.

So... I just want to say about this problem.

**Comment by kolyadkl** [comment]
I found the problem. New liveries from flightsim.to for the FBW.

**Reply by einflori** [reply]
did you experience which one lead to crash? I also have this issue.

**Comment by kolyadkl** [comment]
Its liveries from https://flightsim.to/. I am not sure with one. I found problem with airasia, aeroflot. I think we should wait... Del all new liveries. 
--
Отправлено из Mail.ru для Android понедельник, 05 апреля 2021г., 19:46 +03:00 от einflori  ***@***.*** :
